config/initializers/rack_attack.rb

```diff
@@ -1,71 +1,9 @@
 ##
-# See https://github.com/kickstarter/rack-attack/blob/master/docs/example_configuration.md
-# for more configuration options
+# Throttling is mostly controlled by bot_challenge_page; see its initializer
+# for more details.
 class Rack::Attack
-  ### Throttle Spammy Clients ###
-
-  # If any single client IP is making tons of requests, then they're
-  # probably malicious or a poorly-configured scraper. Either way, they
-  # don't deserve to hog all of the app server's CPU. Cut them off!
-  #
-  # Note: If you're serving assets through rack, those requests may be
-  # counted by rack-attack and this throttle may be activated too
-  # quickly. If so, enable the condition to exclude them from tracking.
-
-  if Settings.THROTTLE_TRAFFIC
-    # Throttle all requests by IP (60rpm)
-    #
-    # Key: "rack::attack:#{Time.now.to_i/:period}:req/ip:#{req.ip}"
-    throttle('req/ip', limit: 300, period: 5.minutes) do |req|
-      req.ip unless req.path.start_with?('/assets') || req.params['format'] == 'json'
-    end
-
-    # Throttle search requests by IP (15rpm)
-    Rack::Attack.throttle('req/search/ip', limit: 15, period: 1.minute) do |req|
```

Comment:
This IP-based throttling is part of `bot_challenge_page` by default; it [groups by subnet](https://github.com/samvera-labs/bot_challenge_page/blob/721489169ef4b9d8510509a514976fd638f1b70f/app/models/bot_challenge_page/config.rb#L69-L80) to decide how to throttle.